demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -125,14 +127,14 @@ def is_builtin_condition_task_branches_handled(self, task: Dict) -> bool:
         for condition in task.get('conditions', []):
             label = condition.get('label')
             if label:
-                task_condition_labels.add(label.upper())
+                task_condition_labels.add(str(label).upper())
```

Comment:
verify that it works as expected, and if it does add comment why the str cast is needed

Reply:
Works when consistent- meaning either fixed to be **'yes'** instead of **yes** in all places or not fixed in all places.
I think we don't have a reason to think it will be partially fixed nowhere because before this fix it would have been failed.

To sum it all- I think it's fine.